src/V3Param.cpp

```diff
@@ -768,7 +768,8 @@ class ParamProcessor final {
                 }
             }
         } else if (AstParamTypeDType* const modvarp = pinp->modPTypep()) {
-            AstNodeDType* const exprp = VN_CAST(pinp->exprp(), NodeDType);
+            AstNodeDType* in_typep = VN_CAST(pinp->exprp(), NodeDType);
+            AstNodeDType* const exprp = in_typep ? in_typep->skipRefToEnump() : nullptr;
```

Comment:
Generally we use camelCase.

```suggestion
            AstNodeDType* rawTypep = VN_CAST(pinp->exprp(), NodeDType);
            AstNodeDType* const exprp = rawTypep ? rawTypep->skipRefToEnump() : nullptr;
```